src/Storage/Exceptions.test.ts

```diff
@@ -0,0 +1,148 @@
+import { pick } from 'ramda'
+import { describe } from 'riteway'
+import '../Extensions/Error'
+import { FetchErrorType } from '../Helpers/FetchError'
+import { FailureReason } from './DownloadFailure'
+import {
+  NoMoreEntriesException,
+  InvalidClaim,
+  IPFSGenericError,
+  IPFSTimeoutError,
+  errorToIPFSError,
+} from './Exceptions'
+
+class FetchError extends Error {
+  readonly type: string
+
+  constructor() {
+    super()
+    this.name = 'FetchError'
+    this.type = FetchErrorType.BodyTimeout
+  }
+}
+
+describe('Exceptions NoMoreEntriesException', async (should: any) => {
+  const { assert } = should('')
+
+  {
+    const noMoreEntriesException = new NoMoreEntriesException('NoMoreEntriesException')
+
+    assert({
```

Comment:
This test is redundant — you have just declared `noMoreEntriesException` to be an instance of `NoMoreEntriesException` in the test.